lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/SimpleModelTest.java

```diff
@@ -16,7 +16,6 @@
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.util.Collections;
-import java.util.List;
```

Comment:
Can you put back the import for `java.util.List`?
It's giving a compilation error on merge with the master branch:
```bash
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/jenkins/agent/workspace/LemMinX-Maven_PR-67-merge/lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/SimpleModelTest.java:[152,17] cannot find symbol
  symbol:   class List
  location: class org.eclipse.lemminx.maven.test.SimpleModelTest
[ERROR] /home/jenkins/agent/workspace/LemMinX-Maven_PR-67-merge/lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/SimpleModelTest.java:[167,17] cannot find symbol
  symbol:   class List
  location: class org.eclipse.lemminx.maven.test.SimpleModelTest
```